internal/collectors/exporter.go

```diff
@@ -0,0 +1,72 @@
+/*
+Copyright 2020 Red Hat, Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package collectors
```

Comment:
Why collectors? Were you thinking of having other metrics here in this pkg?

Reply:
Yes I was thinking of having a collector handling metrics about the exporter and one handling Kubernetes Events

Reply:
To me, those are still collectors used by the exporter in the end, so I find it meaningful to have them under the collectors' package. However, I understand your point here and also feels like having both collectors in the same package would be quite weird.

I see two solutions there:

- Move the exporter collectors inside a `exportercollector` package
- Let the exporter collector in the internal/collector package and implement the event collector in the pkg/collector package

wdyt?